### DISCUSSION TITLE
superlattice

### DISCUSSION THREAD

**Discussion by koce98** [phoebe-team/phoebe#163] [Q&A] [answered]
Dear developers,
Thank you for this nice tool.

Is it possible to compute some properties of superlattices? 
For example, the dependence of superlattice thermal conductivity on interface density, and phonon coherence .

Thanks.
koce

**Comment by jcoulter12** [comment, marked as answer]
Hi Koce, 

Glad you're enjoying Phoebe!

I think there are a few ways you could approach this problem using Phoebe. 

First, if you mean something like what was done in this manuscript --
https://www.science.org/doi/10.1126/science.1225549

It appears they described scattering in the structure using anharmonic scattering for the bulk structure and mass disorder/isotope scattering to represent the effect of the interface. You can see how they describe interfacial scattering (eq 3 in their supplement) is the same as our expression for phonon-isotope scattering here in our theory docs: 
https://phoebe.readthedocs.io/en/develop/theory/phononBTE.html#phonon-scattering-rates
which can be set with this variable (as well as a few listed just below it)
https://phoebe.readthedocs.io/en/develop/inputFiles.html#withisotopescattering

You could apply these two scattering rates to calculate transport using Phoebe, so long as you plan to model interfacial scattering this way. 

Alternatively, there might be a way to do this by training an ML force field and generating anharmonic phonon force constants, as done in this tutorial: 
https://phoebe.readthedocs.io/en/develop/tutorials/mlPhononTransport.html
Though for this, you must consider how difficult it will be to train force fields for your structure, and also think if the interfacial scattering will be captured correctly. 

Let me know if you have any other questions, 
Jenny Coulter

**Reply by koce98** [reply]
Thank you for your reply. I will study it carefully
